Add Colonizers

sibartlett opened the pull request:
A desktop version of [Colonizers](http://colonizers.github.io/).

sindresorhus commented:
Thanks for the suggestion. Before this is added I'd like to see the readme and packaging improved. Readme could use a screenshot and description, and there needs to be a prepackaged binary, so users can just download the app and run it right away => https://github.com/maxogden/electron-packager

Let me know when that's done, and I'll reopen this.

sibartlett commented:
Thanks for the feedback!

I've updated the readme - it now has a better description and a screenshot. I've added downloadable Linux and Mac binaries to the GitHub releases page, via an automated Travis build. I'll look into adding a Windows build when I get a chance.

sindresorhus commented:
Cool, added :)
